recipes/lytekit/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "lytekit" %}
+{% set version = "0.15.27" %}
+{% set python_max = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lytekit-{{ version }}.tar.gz
+  sha256: bf3d9e127ad02b94e88c9b7f4f8df2e81946104c470c935d00ad99b312f46915
+
+build:
+  skip: true  # [py<37 or py>310]
```

Comment:
Drop the selector and instead [use `python_min` variable](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python):

```suggestion
```